packages/xdl/src/project/__tests__/createBundlesAsync-test.ts

```diff
@@ -0,0 +1,61 @@
+import { getConfig, isLegacyImportsEnabled, ProjectTarget } from '@expo/config';
+import { bundleAsync } from '@expo/dev-server';
+
+import { getLogger } from '../ProjectUtils';
+import { createBundlesAsync } from '../createBundlesAsync';
+
+jest.mock('@expo/dev-server');
+jest.mock('@expo/config');
+jest.mock('../../tools/resolveEntryPoint');
+jest.mock('../../project/projectUtils');
```

Comment:
Automated tests are failing because this path cannot be found.

Reply:
I think this was related to a case sensitivity issue- it should be `ProjectUtils`. (It was passing for me locally with the improper case because MacOS is case insensitive while linux/CI is not)